alf/utils/common.py

```diff
@@ -528,20 +528,17 @@ def write_config(root_dir):
     root_dir = os.path.expanduser(root_dir)
     alf_config_file = os.path.join(root_dir, ALF_CONFIG_FILE)
     os.makedirs(root_dir, exist_ok=True)
-    conf_params = getattr(flags.FLAGS, 'conf_param', None)
+    pre_configs = alf.config_util.get_handled_pre_configs()
     config = ''
-    if conf_params:
+    if pre_configs:
         config += "########### config from commandline ###########\n\n"
```

Comment:
Is the message in L534 still valid after the change?

Reply:
> Is the message in L534 still valid after the change?

corrected